apps/api/src/app/events/dtos/trigger-event-request.dto.ts

```diff
@@ -14,6 +14,8 @@ export class SubscriberPayloadDto {
   phone?: string;
   @ApiProperty()
   avatar?: string;
+  @ApiProperty()
+  locale?: string;
```

Comment:
I wonder if there is somewhere an typescript enum with all available locales so we can typesafe this one. Are you familiar with any?

Reply:
I've never used any typescript enum since I didn't find anything appropriate. But I would be glad to improve if anyone knows it 😊

Reply:
That's fine, we can always add this later 🎉